04-Enumerable/Enumerable/Task.cs

```diff
@@ -209,7 +226,8 @@ public class Task {
         /// </example>
         public string GetFirstContainsFirst(IEnumerable<string> data) {
             // TODO : Implement GetFirstContainsFirst
-            throw new NotImplementedException();
+
+	        return data.FirstOrDefault(n => n != null && n.ToUpperInvariant().Contains("FIRST"));
```

Comment:
if you want to compare case insensitively use equals method with proper comparer